deepreg/model/backbone/global_net.py

```diff
@@ -81,18 +79,16 @@ def call(self, inputs, training=None, mask=None):
         :return:
         """
         # down sample from level 0 to E
-        h_in = inputs
         for level in range(self._extract_max_level):  # level 0 to E - 1
-            h_in, _ = self._downsample_blocks[level](inputs=h_in, training=training)
+            h_in, _ = self._downsample_blocks[level](inputs=inputs, training=training)

```

Comment:
is this change valid? as `h_in` is changing in the loop, @YipengHu maybe can you verify?

Reply:
Sorry that's a mistake. You can discard this change.

Reply:
lol how can i remove this? I'm not sue i can do this in GitHub pages, maybe can you add a commit to fix it? thx!

Reply:
Will do !

Reply:
@mathpluscode  I just fixed global_net with a commit